### PR TITLE
feature: quick join

### PR DESCRIPTION
**Pull request by fuwn**
@Bios-Marcel [ref](https://github.com/scribble-rs/scribble.rs/issues/165).

### CODE REVIEW

File: communication/templates/lobby_create.html [hunk 1]
```diff
@@ -97,7 +97,8 @@ <h1>Scribble.rs</h1>
     <div id="join-lobby" class="tab-content">
         <div class="join-lobby-data">
             <div class="table-wrapper-wrapper">
-                <button id="refresh-button" onclick="loadLobbyTable()">Refresh</button>
+	            <button id="quick-join-button" onclick="quickJoinLobby()">Quick Join</button>
```
Comment (fuwn) [inline communication/templates/lobby_create.html:100]: I'm not that great with Go templates but if someone could add a feature where if the `lobbyLength` array is empty, then disabled this button.

**Comment by Bios-Marcel** [comment]
So, the issue wasn't really well defined. I think this feature require a bit more to be useful.

1. You gotta determine the language beforehand, otherwise you'll just end up in lobbies with languages you can't understand.
2. This should be an endpoint, for simplicity. There's more than one client, we are working on a mobile client right now, so that'll be useful there too.

So, I haven't actually thought about the UI for this. I was wondering whether this should just be another tab where you can define your preferences. On top of that "just a random lobby" won't do. You'll probably prefer a lobby that hasn't progressed far and isn't too empty.

Should've written this into the issue, sorry.

File: communication/templates/lobby_create.html (hunk 1 above)
Comment (Bios-Marcel) [reply communication/templates/lobby_create.html:100]: I guess it'd be fine for this to always be enabled and dynamically react depending on whether there's a suitable lobby or not.

**Comment by fuwn** [comment]
Understood, I'll look into further soon. Hopefully maybe even flesh out a preferences tab.

**Comment by Bios-Marcel** [comment]
I think for the future I'll make some system to streamline contributions. We need to communicate before peopke start writing code, because otheewise both parties just waste their time. I'll document more on this tomorrow.

**Comment by Bios-Marcel** [comment]
K, i defined this a bit better.

Either way, I think as of now, the feature won't be so useful, as we don't have a big player base and many people just self-host.
From time to time there's like 20 players at once, but super rarely. I doubt anyone is gonna make much use of this.

If you want to try and put time into this anyways, make a new PR trying to follow the needs described in the issue. Feel free to write a mockup first or give critique on what i wrote in the issue. I'll be closing this PR for now.

The issue: https://github.com/scribble-rs/scribble.rs/issues/165

**Pull request closed without merging**